.ci/pipeline_integration_test

```diff
@@ -0,0 +1,195 @@
+############################################## <Color> ##############################################
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+############################################## </Color> ##############################################
+
+mkdir -p dev
+
+logs_path=.ci/controllers-test/logs
```

Comment:
make a logs folder at this path like done for other providers

Reply:
we already have a logs folder

Reply:
okay